transform/settings.py

```diff
@@ -4,9 +4,10 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-transform-cs: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
+if not os.getenv("CF_DEPLOYMENT", False):
```

Comment:
Same here, these still need to be defined in CF. They just get there values from the manifest